common/smoke-tests/smoke-test-job.yml

```diff
@@ -8,7 +8,7 @@ parameters:
     default: 0
   - name: Timeout
     type: number
-    default: 15
+    default: 30
```

Comment:
Any reason to not use the default of 60 minutes?

Reply:
Not sure. 30 seems to be failing now too. These tests used to take 10 minutes, so I'm not sure what changed in the last few weeks.